src/entity/blog.entity.ts

```diff
@@ -0,0 +1,37 @@
+import { Entity, PrimaryGeneratedColumn, Column, CreateDateColumn, ManyToOne, JoinColumn, BaseEntity } from 'typeorm'
+import User from './user.entity'
+
+@Entity()
+export default class Blog extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  id: number
+
+  @Column()
+  user_id: number
```

Comment:
```suggestion
  userId: number
```
-> Dùng cammel case em nha, e fix các field khác theo style này nhé